core/pta/stats.c

```diff
@@ -37,6 +38,7 @@
  * uint32_t    Biggest byte size which allocation succeeded
  */
 #define STATS_CMD_TA_STATS		3
+#define STATS_CMD_SECTIME_STATS		4
```

Comment:
I suggest to rename the command to `STATS_CMD_SYSTEM_TIME`
Please add an inline description of the ABI, e.g.:
```c
/*
 * STATS_CMD_SYSTEM_TIME - Get system times
 * 
 * [out]    value[0].a        REE time as seen by OP-TEE in seconds
 * [out]    value[0].b        REE time as seen by OP-TEE, milliseconds part
 * [out]    value[1].a        TEE system time in seconds
 * [out]    value[1].b        TEE system time, milliseconds part
 */
#define STATS_CMD_SECTIME_STATS		4
```

Reply:
good points, thanks.